io.github.Soundux.yml

```diff
@@ -0,0 +1,27 @@
+app-id: io.github.Soundux
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: soundux
+finish-args:
+  - --socket=pulseaudio
+  - --socket=x11
+  - --share=ipc
+  - --filesystem=home
+  - --filesystem=host:ro
```

Comment:
Just to double check, I understand there's no way to make it use portals?

Reply:
I'd limit it to xdg-music, xdg-download, /media and /run/media then, if it makes sense from your perspective.

Reply:
Okay, sounds good. And for the write permissions to the `.config/Soundux` folder?

Reply:
I updated the filesystem permissions. I have to check if I can use portals in QtQuick.

Reply:
Are these okay now?